src/components/AppBar/AppBar.jsx

```diff
@@ -43,14 +44,18 @@ const styles = (theme) => ({
   },
 });
 
-const AppBar = ({
-  isLoggedIn, selected, classes, routes,
-}) => {
+// eslint-disable-next-line
+const AppBar = ({ isLoggedIn, selected, classes, routes }) => {
   const [showDrawer, setShowDrawer] = useState(false);
   const pageTitle = (routes.find((item) => item.key === selected) || {}).label;
   const history = useHistory();
 
-  const showDrawerItems = () => (
+  const goBack = () => {
+    history.goBack();
```

Comment:
What happens if there is no history here? For e.g. if the user refreshes the search results page with the search terms in it?

Reply:
Right now, when I refresh the page after the search results and then go back it still returns to the original search results page. I haven't figured out how to post videos in github but I can post one in slack :)